src/main/java/com/minecolonies/achievements/ModAchievements.java

```diff
@@ -0,0 +1,128 @@
+package com.minecolonies.achievements;
+
+import com.minecolonies.MineColonies;
+import com.minecolonies.blocks.ModBlocks;
+import com.minecolonies.items.ModItems;
+import com.minecolonies.lib.Constants;
+import net.minecraft.init.Items;
+import net.minecraft.stats.Achievement;
+import net.minecraftforge.common.AchievementPage;
+
+/**
+ * Achievement collection
+ *
+ * @author Isfirs
+ * @since 0.2
+ */
+public final class ModAchievements
+{
+
+    public static final int ACHIEVEMENT_SIZE_SETTLEMENT = 5;
+    public static final int ACHIEVEMENT_SIZE_TOWN = 10;
+    public static final int ACHIEVEMENT_SIZE_CITY = 20;
+
+    // Achievements
+    /**
+     * Place a supply chest
+     */
+    public static final Achievement achievementGetSupply      = new MineColoniesAchievement("supply", "supply", -2, -2, ModItems.supplyChest, null).registerStat();
+    /**
+     * Use the building tool
+     */
+    public static final Achievement achievementWandOfbuilding = new MineColoniesAchievement("wandofbuilding", "wandofbuilding", 0, -2, ModItems.buildTool, achievementGetSupply).registerStat();
+
+    // Buildings
+    /**
+     * Place a townhall
+     */
+    public static final Achievement achievementBuildingTownhall = new MineColoniesAchievement("townhall", "townhall", -2, 0, ModBlocks.blockHutTownHall, achievementGetSupply).registerStat();
+
+    /**
+     * Upgrade a builder to lv 1
+     */
+    public static final Achievement achievementBuildingBuilder   = new MineColoniesAchievement("upgrade.builder.first", "upgrade.builder.first", 0, 1, ModBlocks.blockHutBuilder, achievementBuildingTownhall).registerStat();
+    /**
+     * Max out a builder
+     */
+    public static final Achievement achievementUpgradeBuilderMax = new MineColoniesAchievement("upgrade.builder.max", "upgrade.builder.max", 2, 1, ModBlocks.blockHutBuilder, achievementBuildingBuilder).registerStat();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 217 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)